cypress/support/step-definitions/fieldset-steps.js

```diff
@@ -0,0 +1,17 @@
+import { legendInput, legendPreview } from '../../locators/legend';
+
+When('I set legend to {string}', (legend) => {
+  legendInput().clear().type(legend);
```

Comment:
maybe we should aggregate those functions as `clear.type` into helper to avoid copy code ?

Reply:
Our approach is to handle fluent interface element.method().otherMethod() etc. instead of method(element)